dandiapi/api/views/info.py

```diff
@@ -0,0 +1,15 @@
+from django.conf import settings
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+
+schema_url = f'https://raw.githubusercontent.com/dandi/schema/master/releases/{settings.DANDI_SCHEMA_VERSION}/dandiset.json'
```

Comment:
This is a bit smelly, but IMO adding another environment variable for this wouldn't really be worth it, as I don't expect this URL to change, and it would add more complexity.

Reply:
I think it's fine, since this is derived from another environment variable anyway.